src/libraries/System.Private.CoreLib/src/System/OperatingSystem.cs

```diff
@@ -219,7 +218,6 @@ public static bool IsMacOSVersionAtLeast(int major, int minor = 0, int build = 0
         /// </summary>
```

Comment:
We should update the summary text here to make it clear that we're not using `libobjc.get_operatingSystemVersion` to get the host macOS version but returning the actual iOS support version that MacCatalyst targets.

Reply:
Mac Catalyst is not considered macOS so it returns `false` for `IsMacOS`/`IsMacOSVersionAtLeast`.

Reply:
...but yeah, I see I left the same comment on the Catalyst branch.